gnuradio-runtime/include/gnuradio/buffer_single_mapped.h

```diff
@@ -33,7 +33,7 @@ class GR_RUNTIME_API buffer_single_mapped : public buffer
     gr::logger_ptr d_logger;
     gr::logger_ptr d_debug_logger;
 
-    virtual ~buffer_single_mapped();
```

Comment:
After reading up on this detail, it looks like `virtual` is propagated implicitly from the base class and it therefore not strictly needed here.  I guess that's a detail I'm just not used to yet.